abdiff/core/utils.py

```diff
@@ -48,3 +48,14 @@ def update_or_create_job_json(job_directory: str, new_data: dict) -> dict:
 def update_or_create_run_json(run_directory: str, new_data: dict) -> dict:
     """Create or update a run's JSON file."""
     return update_or_create_json(run_directory, "run.json", new_data)
+
+
+def create_subdirectories(
+    base_directory: str, subdirectories: list[str]
+) -> tuple[str, ...]:
```

Comment:
I think a tiny docstring here could be helpful, touching on _why_ this is helpful  (i.e. some functions have to create a few subdirectories, and this normalizes an approach).

Reply:
Docstring has been added.